src/include/concurrency/transaction_context.h

```diff
@@ -89,13 +89,13 @@ class TransactionContext : public Printable {
   }
 
   void RecordCreate(oid_t database_oid, oid_t table_oid, oid_t index_oid) {
-    rw_object_set_.emplace_back(database_oid, table_oid, index_oid,
-                                DDLType::CREATE);
+    rw_object_set_.push_back(std::make_tuple(database_oid, table_oid,
```

Comment:
`concurrent_vector` also has a `emplace_back` method. Is there a reason to use `push_back(std::make_tuple(...))` instead?

Reply:
`emplace_back` is only supported in c++11. It compiled fine on my machine but didn't build on travis. So I moved to `push_back`.